src/pretix/presale/templates/pretixpresale/organizers/index.html

```diff
@@ -147,11 +139,11 @@ <h3>{% trans "Upcoming events" %}</h3>
         {% empty %}
             {% if "old" in request.GET %}
                 <div class="col-md-12">
-                    <em>{% trans "No archived events found." %}</em>
+                    <p><em>{% trans "No archived events found." %} <a href="?{% url_replace request "old" "" %}">{% trans "Show upcoming" %}</a></em></p>
                 </div>
             {% else %}
                 <div class="col-md-12">
-                    <em>{% trans "No public upcoming events found." %}</em>
+                    <p><em>{% trans "No public upcoming events found." %} <a href="?{% url_replace request "old" "1" %}">{% trans "Show past events" %}</a></em></p>
```

Comment:
This change would mean we no longer allow finding past events if there is any upcoming event? I don't think that's a reasonable change without discussion :)

Reply:
You are right, I have not realized this! I moved them outside the empy-part and unified the layout with the other views regarding text-placement. Or do you want this links to be at the top?

![Bildschirmfoto 2021-12-13 um 10 22 28](https://user-images.githubusercontent.com/276509/145785478-8126f6e1-c19f-4ba1-8aef-fadc5231f1a3.png)
![Bildschirmfoto 2021-12-13 um 10 22 52](https://user-images.githubusercontent.com/276509/145785525-cd3bcda8-0d08-4714-bc90-a281bb19957d.png)